next.js

```diff
@@ -0,0 +1,7 @@
+const menu = document.querySelector('#mobile-menu')
+const menuLinks = document.querySelector('.navbar-menu')
+
+menu.addEventListener('click', function() {
+	menu.classList.toggle('is-active');
+	menuLinks.classList.toggle('active');
+})
```

Comment:
Good job with the JS.

Do you understand what's going on here tho. I mean if you can explain it when asked.